tests/manage/pv_services/pvc_resize/test_pvc_expansion_when_full.py

```diff
@@ -0,0 +1,265 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.resources.pod import get_used_space_on_mount_point
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version, ManageTest, tier2, polarion_id
+)
+from ocs_ci.utility.prometheus import PrometheusAPI, check_alert_list
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@skipif_ocs_version('<4.5')
+@polarion_id('OCS-301')
+class TestPvcExpansionWhenFull(ManageTest):
+    """
+    Tests to verify PVC expansion when the PVC is 100% utilized.
+    Verify utilization alert will stop firing after volume expansion.
+
+    """
+    @pytest.fixture(autouse=True)
+    def setup(self, create_pvcs_and_pods):
+        """
+        Create PVCs and pods
+
+        """
+        self.pvc_size = 4
+        self.pvcs, self.pods = create_pvcs_and_pods(
+            pvc_size=self.pvc_size,
+            access_modes_rbd=[constants.ACCESS_MODE_RWO],
+            access_modes_cephfs=[constants.ACCESS_MODE_RWO]
+        )
+
+    def test_pvc_expansion_when_full(self):
+        """
+        Verify PVC expansion when the PVC is 100% utilized.
+        Verify utilization alert will stop firing after volume expansion.
+
+        """
+        pvc_size_expanded = 10
+
+        # Run IO to utilise 100% of volume
+        log.info("Run IO on all to utilise 100% of PVCs")
+        for pod_obj in self.pods:
+            pod_obj.run_io(
```

Comment:
will `rate=200M` fill the volume quicker?

Reply:
While developing another test case, I tried using rate='100M' and observed MDS slow response messages and client IO taking more network bandwidth.
Is rate='100M' an optimal value ? IO will be running on two pods.

Reply:
yes, 100M should be good

Reply:
Thanks !. Added rate='100M'